client/charts/js/lib/utilities.js

```diff
@@ -75,7 +75,7 @@ export function getFilteredUrl(databasePath, filtersApplied, currentDate) {
 		parameters.push(`state=${filtersApplied.state.replace(' ', '+')}`)
 	}
 
-	if (filtersApplied.year !== null && filtersApplied.monthName === undefined) {
+	if (filtersApplied.year !== null && filtersApplied.year !== undefined && filtersApplied.monthName === undefined) {
```

Comment:
I think this is fine, but I do wonder if this whole function `getFilteredUrl` could do with some cleanup for all this null/undefined checking that's going on (something like iterating over fields that _do_ exist or having the `filtersApplied` object know how to convert itself to a URL and have the code live there... just some ideas for future enhancements).

Reply:
Yes. I agree